### PR TITLE
fix: stepsRemaining value of the early adopters plan's input section

### PR DESCRIPTION
**Pull request by anikdhabal**
Before:-

Screenshot 2024-01-03 223532

Now:-

Screenshot 2024-01-04 175322

### CODE REVIEW

**Comment by catalinpit** [comment]
Hey @anikdhabal, thanks for the PR! Could you provide more information about what's this PR changing, please? Also, is there an issue for this or something?

It's always good to provide information about the changes when you open a PR. :-)

**Comment by anikdhabal** [comment]
> Hey @anikdhabal, thanks for the PR! Could you provide more information about what's this PR changing, please? Also, is there an issue for this or something?
> 
> It's always good to provide information about the changes when you open a PR. :-)

Thanks for your review. It was a simple fix.

Before:-
Screenshot 2024-01-03 223532

At first, it showed 1 step remaining to be signed. However, there are a total of 3 steps, and this is the first step.

Now:-

Screenshot 2024-01-04 175322

File: apps/marketing/src/components/(marketing)/widget.tsx [hunk 1]
```diff
@@ -90,10 +91,10 @@ export const Widget = ({ className, children, ...props }: WidgetProps) => {
     }
```
Comment (catalinpit) [inline apps/marketing/src/components/(marketing)/widget.tsx:91]: I'm curious about the reason for using `useMemo` here. Do you think it's necessary?

**Review by dguyen — APPROVED** [review]
Thanks for the catch 😄

**Comment by anikdhabal** [comment]
> Thanks for the catch 😄

thank you too

**Pull request merged**